src/components/screens/DashboardDRE/DashboardDREContainer.jsx

```diff
@@ -53,6 +55,17 @@ class DashboardDREContainer extends Component {
     };
   }
 
+
+  componentDidMount() {
+    getPendentesAprovacaoList().then(response => {
```

Comment:
getPendentesAprovacaoList().then(pendentesAprovacaoList => {
      this.setState({
        pendentesAprovacaoList
      });
    });